.vscode/launch.json

```diff
@@ -67,7 +67,7 @@
                 "MOCHA_enableTimeouts": "0", // Disable time-outs
                 "DEBUGTELEMETRY": "1", // 1=quiet; verbose=see telemetry in console; 0=send telemetry
                 "NODE_DEBUG": "",
-                "DISABLE_LANGUAGE_SERVER_TESTS": "0",
+                "DISABLE_LANGUAGE_SERVER": "0",
```

Comment:
Renamed DISABLE_LANGUAGE_SERVER_TESTS to DISABLE_LANGUAGE_SERVER because it now also controls packaging the server.